recipes/scdoc/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class ScdocInstallerConan(ConanFile):
+    name = "scdoc"
+    description = "scdoc is a simple man page generator for POSIX systems written in C99."
+    topics = ("scdoc", "documentation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://git.sr.ht/~sircmpwn/scdoc"
+    license = "Unknown"
+    settings = "os", "arch", "compiler", "build_type"
+    _autotools = None
+    no_copy_source = True
```

Comment:
```suggestion
```

Only for header-only or very big packages like Boost C++

Reply:
corrected